README.md

```diff
@@ -1,26 +1,105 @@
- 	* Generate a map tile: `/tile/{z}/{x}/{y}.png` serves tiles.
- 	* Generate a UTF grid: `/grid/{z}/{x}/{y}?utfFields=field1,field2,field...N` etc, where a `field` is a column in your data table.
- * To publish, make sure you have specified valid AWS credentials and have listed the proper database credentials in a `.env` file in the root of the project. The format for this file can be copied from `env-template`. You can then run `./scripts/publish` to publish to AWS Lambda.
-    * For subsequent deployments of the same lambdas, leave off the `--new` tag.
-    * **NOTE:** Only `LAMBDA_TIMEOUT` and `LAMBDA_MEMORY` are changed on "update" deployments, all other settings must be changed by logging in to the Lambda Management Console or by deploying a new project.
- * API Gateway has trouble serving images, so you need to configure some sort of proxy (e.g. using a CloudFront distribution) that sets an `Accept:image/png` header on all lambda requests.
+### Local Development
+Dependencies: docker, docker-compose
+ * Clone the Tilegarden repository to your machine.
+ * Add any development data:
+   * Add your geospatial data to the `data/` directory. .zipped shapefiles and gzipped SQL dumps will be loaded in to the development database automatically.
+   * Make sure the name of the file matches the name of the SQL table that contains the data.
+ * Configure your map:
+   * Open the file [`src/tiler/src/config/map-config.mml`](src/tiler/src/config/map-config.mml). This is where you specify your map settings. [A full specification for Carto's .mml format can be found here.](https://cartocss.readthedocs.io/en/latest/mml.html)
+   * Of particular note is the “Layer” property, which specifies the layers of your map (as an array). Odds are you won't have to change the target srs (at the top of the file), but make sure that the srs of each layer is specified properly. By default, all layers of your map are displayed at once, but different combinations of layers can be selected using the `layers` query string. See [Filters](#filters) for more info.
+ * Create a copy of [`env-template`](env-template) named `.env`. This contains production-specific configuration options and doesn't need to be filled out right now (but must exist in order to run the development server).
+ * Run `./scripts/update` to create your Docker containers and populate the development database. The optional flag `--download` will download the data sets used for our demos.
+ * Run `./scripts/server` to start the development server. It will be available at [localhost:3000](http://localhost:3000/), your tiles can be found at `/tile/{z}/{x}/{y}.png`.
+ * The local development server exposes a node.js debugger, which can be attached to by Chrome DevTools or your IDE of choice [(see below)](#Debugging).
+   * (Optional) If you downloaded the example data using `./scripts/update --download`, opening `index.html`(index.html) (or any of the pages in `demo/`) should show you working demos.
```

Comment:
Looks like `index.html` was intended to be in square brackets so it would linkify.